vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/KeyboardSteps.java

```diff
@@ -70,6 +70,18 @@ public KeyboardSteps(KeyboardActions keyboardActions, IBaseValidations baseValid
         this.webDriverProvider = webDriverProvider;
     }
 
+    /**
+     * Type <b>text</b> into the <b>element</b>
+     * @param text text to type into the element
+     * @param locator locator to find an element
+     */
+    @When("I type `$text` in field located `$locator` and keep keyboard opened")
+    public void typeTextInFieldAndKeepKeyboard(String text, Locator locator)
+    {
+        baseValidations.assertElementExists("The element to type text", locator)
+                .ifPresent(e -> keyboardActions.typeTextAndHide(e, text));
```

Comment:
```suggestion
                .ifPresent(e -> keyboardActions.typeText(e, text));
```

and the step below should be updated to use a method with updated name

Reply:
The code has been updated.